.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "go.testEnvVars": {
+        "DB": "postgresql://postgres:toor@localhost:15432",
```

Comment:
Need to use different env vars since we're running tests outside of the Docker stack. So host/ports are the ones external ones.